miscellaneous/fhir-server/scripts/README.md

```
@@ -0,0 +1,31 @@
+To create the necessary database tables in the FHIR repository, you can use the following SQL script. This script will 
+set up the required tables for storing FHIR resources and related metadata.
+
+1. Run the following sql schema script against your database to create the necessary tables based on your Database Type.
+
+    #### For PostgreSQL
+    Run [schema-postgresql.sql](schema-postgresql.sql)
+
+
+2. Then run insert_SEARCH_PARAM_RES_EXPRESSIONS_postgres.sql script to persist data to the SEARCH_PARAM_RES_EXPRESSIONS table.
+
+    #### For SQL Server
```

markdownlint-cli2 (0.22.0)

[warning] Headings must start at the beginning of the line

(MD023, heading-start-left)